documentation/api/query/v4/packages.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: "PuppetDB 4.4: Packages endpoint"
+title: "PuppetDB 4.4: Package endpoints"
 layout: default
 canonical: "/puppetdb/latest/api/query/v4/packages.html"
```

Comment:
We talked about this offline; I think the duplication is silly enough here that it warrants squishing things together. Looked at the site, and I think the presentation will be ok.